prow/lib.sh

```diff
@@ -114,10 +114,10 @@ function build_images() {
   targets="docker.pilot docker.proxyv2 "
 
   # use ubuntu:jammy to test vms by default
-  nonDistrolessTargets="docker.app docker.app_sidecar_ubuntu_jammy docker.ext-authz "
+  nonDistrolessTargets="docker.app docker.app_sidecar_centos_stream_8 docker.ext-authz "
   if [[ "${JOB_TYPE:-presubmit}" == "postsubmit" ]]; then
     # We run tests across all VM types only in postsubmit
-    nonDistrolessTargets+="docker.app_sidecar_ubuntu_xenial docker.app_sidecar_debian_11  docker.app_sidecar_centos_7 "
+    nonDistrolessTargets+="docker.app_sidecar_ubuntu_xenial docker.app_sidecar_ubuntu_jammy docker.app_sidecar_debian_11 docker.app_sidecar_centos_7 "
```

Comment:
Do we need to run this postsubmit target build ? It looks like that's building ubuntu and debian images. Should we skip that ?

Reply:
Actually, we could skip these images... But let's merge it as is for now and we can remove it in a follow-up.